File: Arrays_And_Hashing/1550.Three_Consecutive_Odds.go
func threeConsecutiveOdds(arr []int) bool {
    tsum := 0
    for _,n := range(arr) {
        if n % 2 == 0 {tsum = 0} else {tsum++}
        if tsum == 3 {return true}
    }
    return false
}

File: Arrays_And_Hashing/1550.Three_Consecutive_Odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        tsum = 0
        for n in arr:
            tsum = 0 if n % 2 == 0 else tsum+1
            if tsum == 3: return True
        return False
